tests/json_queries.cc

```diff
@@ -100,6 +100,64 @@ TEST(AddImage, simpleAdd)
 
     EXPECT_EQ(json_response[0]["AddImage"]["status"].asString(), "0");
     VDMSConfig::destroy();
+    PMGDQueryHandler::destroy();
+}
+
+TEST(UpdateEntity, simpleAddUpdate)
```

Comment:
I am inclined to move all this testing to python. 
Having all this tests in python make great tutorials. 
Since we are testing at user-level query (means black-box test), I think it make more sense to just do those in python, and leave the unit testing in C++ (the PMGDQueryHandler testing is a perfect example of unit testing)
We should no longer keep adding stuff to json_queries.cc (which reads the query from a file) and migrate those tests to Python (most of them are already in Python in any case)

Reply:
We will leave this for now, and keep adding Python tests as well.